src/iron_math.py

```diff
@@ -1,16 +1,35 @@
 import math
+import arcade
 from typing import Tuple
 
 
-def rotate_vec2(point: Tuple[float, float], radians: float, /) -> float:
+def rotate_vec(point: Tuple[float, float], radians: float, /) -> float:
     x, y = point
     s = math.sin(radians)
     c = math.cos(radians)
     return (c * x - s * y, s * x + c * y)
 
 
-def add_vec2(point: Tuple[float, float], translation: Tuple[float, float], /) -> float:
+def add_vec(point: Tuple[float, float], translation: Tuple[float, float], /) -> float:
```

Comment:
what's the syntax of the last argument here? That's new to me!

Reply:
It marks every prior argument as positional-only, cannot be passed as `add_vec(point=foo, translation=bar)`

I threw it in without much thought other than a guess that it might be faster, but didn't follow up.  So I googled the PEP, and sure enough, it is faster:

https://peps.python.org/pep-0570/#performance

We could certainly call this a premature optimization, though on the other hand, I'm not sure anyone wants to invoke these functions with named parameters anyway.

If we opt to keep the positional-only marker on these math routines, then I should go in and add it to the others, too.

Reply:
interesting, thanks!  I added an Issue around this so we can have a asynchronous discussion around it: https://github.com/CastIronChat/twisted-metal/issues/38